### DISCUSSION TITLE
出题征集

### DISCUSSION THREAD

**Discussion by uvd** [move-cn/lets-rust#1] [Ideas]
1. hello world / 编辑器
2. 基本类型/struct/enum
3. 循环
4. 定义函数
5. 错误处理Option/Result
6. 集合，vec，hashmap
7. trait/泛型
8. 生命周期 所有权
9. 多线程 & 锁
10. macros 宏
11. serde
12. HTTP Client /  HTTP Server
13. CLI 
14. tokio
15. async并发
16. axum & database
17. grpc
18. Sui SDK CALL
19. 实现简单区块链

**Comment by lispking** [comment]
https://github.com/rust-lang/rustlings/tree/main/exercises
对于初学者来说，这应该是每个rust入门者都要经历的练习之一
